### PR TITLE
- Updating consoles as per discussion in #3 to use simple-console.

### PR DESCRIPTION
**Pull request by kenwheeler**
- Replaced invariants with explicit throws
- Added logic and config to remove consoles/invariants from running in production & standalone

### CODE REVIEW

File: lib/Store.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ var _classCallCheck = function (instance, Constructor) { if (!(instance instance
 
 var EventEmitter = require("events").EventEmitter;
 var assign = require("object-assign");
-var iv = require("invariant");
+var SimpleConsole = require("simple-console");
+var con = process.env.NODE_ENV === "production" ? new SimpleConsole(null) : new SimpleConsole();
```
Comment (ryan-roemer) [inline lib/Store.js]: GLOBAL: I'd move all of this to one file with a singleton to simplify all of this like `lib/console.js`

``` js
module.exports = process.env.NODE_ENV === "production" ? new SimpleConsole(null) : new SimpleConsole();
```

And then elsewhere:

``` js
var con = require('./lib/console');
```

**Comment by kenwheeler** [comment]
Good call

**Comment by ryan-roemer** [comment]
@kenwheeler -- Sorry for the moving target here, but I've slightly refactored how the Noop logger / base class works in `v0.1.0`, which you should upgrade to. Read about it here: https://github.com/FormidableLabs/simple-console#noop-logger

---

### Noop Logger

There are some cases where you will want to conditionally silence the logger.
You can do this by passing setting the `noop` option:

``` js
var con = new SimpleConsole({ noop: true });
con.log("Hello world!"); // => Should _not_ output anything.
```

This is usually useful in a case where you build different capabilities based
on some external information, for example, React-style, this could be
something like:

``` js
var con = new SimpleConsole({
  noop: process.env.NODE_ENV === "production"
});

con.log("Hello world!"); // => Should _not_ output anything in `production`.
```

**Comment by ryan-roemer** [comment]
@kenwheeler -- Ugh just realize the docs are wrong :/ Fixing that...

UPDATE: _Docs / comment above fixed_

**Pull request merged**